# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/client/ClientImageLayer.cpp
# Commit: c09a51622e98
# Full Hash: c09a51622e988e81077c75e3aaa5a49df66284b3
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-22 16:44:08
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 737807563dd5 -r c09a51622e98 gfx/layers/client/ClientImageLayer.cpp
--- a/gfx/layers/client/ClientImageLayer.cpp	Thu Mar 21 23:14:26 2019 +0000
+++ b/gfx/layers/client/ClientImageLayer.cpp	Fri Mar 22 03:15:14 2019 +0000
@@ -120,8 +120,8 @@
     return;
   }
 
-  if (!mImageClient ||
-      !mImageClient->UpdateImage(mContainer, GetContentFlags())) {
+  if (!mImageClient || !mImageClient->UpdateImage(mContainer, GetContentFlags(),
+                                                  Nothing())) {
     CompositableType type = GetImageClientType();
     if (type == CompositableType::UNKNOWN) {
       return;
@@ -137,7 +137,8 @@
       mImageClient->Connect();
       ClientManager()->AsShadowForwarder()->Attach(mImageClient, this);
     }
-    if (!mImageClient->UpdateImage(mContainer, GetContentFlags())) {
+    if (!mImageClient->UpdateImage(mContainer, GetContentFlags(),
+                                   Nothing())) {
       return;
     }
   }